scenarios/04_Battle_of_Barnon.cfg

```diff
@@ -1224,6 +1274,93 @@
         [/if]
     [/event]
 
+    [event]
+        name=enemies defeated
+        first_time_only=no # because this could fire when Hoyre is the only enemy, in which case we need to trigger the betrayal, which means it could happen again
+        [if]
+            [variable]
+                name=ano_barnon_tr
+                equals=no
+            [/variable]
+            [and]
+                [have_unit]
+                    id=Bor Cryne
+                    [filter_side]
+                        [allied_with]
+                            side=1
+                        [/allied_with]
+                    [/filter_side]
+                [/have_unit]
+            [/and]
+            [and]
+                [have_unit]
+                    id=Uri van Roe
+                    [filter_side]
+                        [allied_with]
+                            side=1
+                        [/allied_with]
+                    [/filter_side]
+                [/have_unit]
+            [/and]
+            [then]
+                {MSG_Uri _"...well, now that Gawen thinks he's won, there's no need to keep up this farce, is there, Bor?"}
+                [modify_side]
+                    side=4
+                    team_name=bad
+                [/modify_side]
+
+                {REPLACE_SCENARIO_MUSIC theme_of_a_new_order.ogg}
+                {MSG_Cryne _"Indeed, Uri, there isn't! Let our swords instead cut that underling's head off! And slay that witch, his step-mother, as well!"}
```

Comment:
Bor Cryne probably talks in a little rougher way than this, but if you don't feel like rewriting it we can leave it for now.

Reply:
which part, the "Indeed"? How about replacing that with "Damn straight"?

Reply:
The overall diction and flow of the sentence. Style isn't really that scientific, it just doesn't sound like Bor. It's not really a big deal though.

Reply:
ok well I dunno how to change it, then... 
(note that I just copied this from where I'd added it previously in PR #53 to add the NIGHTMARE difficulty; it was at line 1948 then but is now at line 2106)

Reply:
Yeah this is how our conversation about it went previously: <img width="830" alt="Screen Shot 2022-08-03 at 11 54 42" src="https://user-images.githubusercontent.com/1388660/182654218-b4330695-25a4-4eb1-8185-1ccdc18cd2da.png">